src/routes/locking/locking.controller.spec.ts

```diff
@@ -179,8 +179,8 @@ describe('Locking (Unit)', () => {
           previous: null,
           results: campaignsPage.results.map((campaign) => ({
             ...campaign,
-            periodStart: campaign.periodStart.toISOString(),
-            periodEnd: campaign.periodEnd.toISOString(),
+            startDate: campaign.startDate.toISOString(),
+            endDate: campaign.endDate.toISOString(),
             lastUpdated: campaign.lastUpdated.toISOString(),
```

Comment:
What do you think about making a `toJson` helper for these (we have a lot for different transaction types)? e.g.

```ts
function toJson(campaign: Campaign): unknown {
  ...campaign,
  periodStart: campaign.periodStart.toISOString(),
  periodEnd: campaign.periodEnd.toISOString(),
  lastUpdated: campaign.lastUpdated.toISOString()
}
```